packages/utilities/src/array.ts

```diff
@@ -76,6 +76,6 @@ export function toMatrix<T>(items: T[], columnCount: number): T[][] {
  * @param array - The array to operate on
  * @param index - The index of the element to remove
  */
-export function removeElement<T>(array: T[], index: number): T[] {
+export function removeIndex<T>(array: T[], index: number): T[] {
```

Comment:
Breaking change, I know, I'm the worst. But I just added this api late last week, so it's unlikely anyone has taken a dependency on it yet. I also think aligning the names with findIndex is a worthwhile change.

Reply:
Hm... why is this API better than the native `splice`?

Reply:
It doesn't mutate the original array